README.md

```diff
@@ -39,13 +39,13 @@ Input:
 Output:
 ```ts
 export interface ExampleSchema {
-  firstName: string;
-  lastName: string;
+  "firstName": string;
```

Comment:
**Pros**

\* ?

**Flaws** 

\* More potential bugs:
&nbsp;&nbsp;&nbsp;&nbsp; $ — valid
&nbsp;&nbsp;&nbsp;&nbsp; \_0 — valid
&nbsp;&nbsp;&nbsp;&nbsp; 0\_ — invalid
&nbsp;&nbsp;&nbsp;&nbsp; undefined — invalid

\* Worse performance
\* More tests
\* Inconsistent view:

```ts
interface X {
   "0_": string;
   _0 : string;
}
```

Why do you want these changes for auto generated code?